src/main/webapp/app/overview/exercise-details/exercise-details-student-actions.component.ts

```diff
@@ -143,8 +143,8 @@ export class ExerciseDetailsStudentActionsComponent {
      * @return {assignedTeamId}
      */
     get assignedTeamId(): number | undefined {
-        const participation = this.exercise.studentParticipations![0];
-        return participation ? participation.team?.id : this.exercise.studentAssignedTeamId;
+        const participation = this.exercise.studentParticipations;
+        return participation && participation?.length > 0 ? participation[0].team?.id : this.exercise.studentAssignedTeamId;
```

Comment:
Fwiw, you don't need the `?` after `participation` any more if you already account for it being undefined/null in the check before, but it doesn't hurt either in the end.